tool/canonical/fix_excludes.json

```diff
@@ -0,0 +1,94 @@
+[
+  {
+    "lint" : "avoid_renaming_method_parameters",
+    "notes" : "naming"
+  },
+  {
+    "lint" : "avoid_shadowing_type_parameters",
+    "notes" : "naming"
+  },
+  {
+    "lint" : "camel_case_extensions",
+    "notes" : "naming"
+  },
+  {
+    "lint" : "camel_case_types",
+    "notes" : "naming"
+  },
+  {
+    "lint" : "constant_identifier_names",
+    "notes" : "naming"
+  },
+  {
+    "lint" : "control_flow_in_finally",
+    "notes" : "TODO"
```

Comment:
I'm not sure what "TODO" means. If it mean "figure out whether we can fix it", then I don't think we can.

Reply:
👍